main.py

```diff
@@ -5,13 +5,80 @@
 if __name__ == "__main__":
     parser = argparse.ArgumentParser()
     parser.add_argument("--path", "-p", help="set the path to the c files")
+    parser.add_argument("--graph", action='store_const', const=True, default=False, help="generate a graph image")
+    parser.add_argument("--output", "-o", help="the file where c files will be genreated")
     
     args = parser.parse_args()
-    
-    graph = extract.generate_graph(args.path)
-    
-    bc = graph.biconnected_components()
-    igraph.plot(graph, target="bc.svg", bbox=(0, 0, 9200, 9200), vertex_label=graph.vs["short"], mark_groups=bc)
-    
-    ceb = graph.community_edge_betweenness(directed=False)
-    igraph.plot(graph, target="ceb.svg", bbox=(0, 0, 9200, 9200), vertex_label=graph.vs["short"], mark_groups=ceb.as_clustering())
+    if args.graph:
+        graph = extract.generate_graph(args.path)
+
+        bc = graph.biconnected_components()
+        igraph.plot(graph, target="bc.svg", bbox=(0, 0, 9200, 9200), vertex_label=graph.vs["short"], mark_groups=bc)
+
+        ceb = graph.community_edge_betweenness(directed=False)
+        igraph.plot(graph, target="ceb.svg", bbox=(0, 0, 9200, 9200), vertex_label=graph.vs["short"], mark_groups=ceb.as_clustering())
+    else:
+        """
```

Comment:
Probably should be a `#` comment rather than a sort-of doc string?

Reply:
From my understanding """ are the only way to create mulltiline comments and the only official one, but that shouldn't be a problem.

Reply:
`"""` in this sense is for doc comments for methods (etc).: for "normal" comments you just have to use `#` repeated for each line, I'm afraid.

Reply:
Yes this is how I did it in the end.

https://github.com/capablevms/function-community-finding/blob/c597ed8134fe6b07a4a40eec7d5606d508704037/main.py#L21-L23